apps/condo/domains/banking/constants.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const NESTED_SET_UTIL = 'nested-set-util'
+
+const DV_SENDER_FOR_NESTED_SET_UTIL = {
```

Comment:
it's better to move close to usage. it does not look like domain consts

Reply:
Moved